weave-js/src/components/PagePanelComponents/Home/Browse2/SmallRef.tsx

```diff
@@ -76,8 +78,93 @@ export const objectRefDisplayName = (
   throw new Error('Unknown ref type');
 };
 
-export const SmallRef: FC<{
```

Comment:
the biggest number of line changes of this PR are in this file. here I just wanted to separate the `SmallRef` for weave-objects and for artifacts into separate components. 

to do that and avoid code duplication, I moved the "item" into this `SmallRefBox` component